src/test/java/seedu/address/logic/commands/DeleteLessonCommandTest.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.calendar.Lesson;
+
+public class DeleteLessonCommandTest {
+
+    @Test
+    public void deleteValidLesson() throws CommandException {
+        Lesson lesson = new Lesson("Math", DayOfWeek.MONDAY, LocalTime.of(12, 0), LocalTime.of(13, 0));
+        Model model = new ModelManager();
+        model.addLesson(lesson);
+
+        DeleteLessonCommand command = new DeleteLessonCommand("Math");
+        CommandResult result = command.execute(model);
+        assertEquals(result.getFeedbackToUser(),
+                String.format(DeleteLessonCommand.MESSAGE_DELETE_STUDENT_SUCCESS, lesson));
+    }
+
+    @Test
+    public void deleteNonExistentLesson() {
```

Comment:
could you check if this meets the test naming conventions?
![image](https://github.com/user-attachments/assets/f764cc63-89fb-4744-aa90-c5c49b102e81)
https://se-education.org/guides/conventions/java/intermediate.html

Reply:
(Josh caught my one LOL)

Reply:
tp code also never use underscore, should be okay

Reply:
ok I will change it

Reply:
I have change it